pyVMC.py

```diff
@@ -2189,18 +2190,133 @@ def getSDDCPublicIP(proxy_url, sessiontoken):
 # NSX-T - Segments
 # ============================
 
-def configure_t1(proxy, session_token, t1_id, connection_type):
+def configure_t1(**kwargs):
     """ Configures a Tier1 router as 'ROUTED', 'ISOLATED', or 'NATTED'... Creates a new T1 if it does not exist already."""
-    json_data = {"type": connection_type}
+    if kwargs['objectname'] is None:
+        print("Please use -n to specify the name of the T1 router to be configured.  Consult the help for additional options.")
+        sys.exit()
+    t1_id = kwargs["objectname"]
+    json_data = {"type": kwargs["t1type"]}
```

Comment:
We should do some argument validation since a user mistake in t1 type will result in an API failure.